src/components/MainLayout/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { PageProps } from 'gatsby'
 import ThemeMainLayout, {
-  LayoutModifiers,
+  // LayoutModifiers,
```

Comment:
Is there reason for this being commented out instead of deleted?

Reply:
I dropped the comment. Thanks.